stable/insights-agent/ci/test-values.yaml

```diff
@@ -149,6 +149,20 @@ awscosts:
   tagvalue: staging.internal.reactiveops.com
   workgroup: cur_athena_workgroup
 
+cloudcosts:
+  enabled: false
+  secretName: cloudcosts-secret
+  tagkey: kubernetes_cluster
+  tagvalue: staging.internal.reactiveops.com
+  aws:
+    awsAccessKeyId: someAccessID
```

Comment:
You don't need `aws` prefixing here due to context

Reply:
right